libqaul/src/api/mod.rs

```diff
@@ -35,26 +37,48 @@ impl Qaul {
     /// Generates a new `Identity` and takes a passphrase that is used
     /// to encrypt
     pub fn user_create(&self, _pw: &str) -> QaulResult<UserAuth> {
-        let user = _User::new();
+        let user = User::new();
         let id = user.id.clone();
         let mut users = self.users.lock().unwrap();
         users.insert(id.clone(), user);
 
-        Ok(UserAuth::Default(id))
+        Ok(UserAuth::Trusted(id, "fixme".into())) // TODO: Actually generate passphrase
```

Comment:
I'd rather this line was a call to an `unimplemented!()` function in the `auth` module. We should let unimplemented things crash, rather than have "fixme" values :sweat_smile:

Reply:
I chose not to do that because it would cause every test to autofail - if that's what we want, maybe I can make the unimplemented bit `#![cfg(not(test))]`? That feels wrong, somehow.

Reply:
Oh, or, another option: we could have some way of injecting a user with the desired key material into the `Qaul` directly. I'll prototype that and see how well it works.

Reply:
This is done in a714d91 and c689a18